### DISCUSSION TITLE
Record configuration per camera

### DISCUSSION THREAD

**Discussion by Deishelon** [blakeblackshear/frigate#6961] [Ideas]
Hi,

Right now my record (https://docs.frigate.video/configuration/record) configuration is set to save last 10 days of footage (mode all)

However, it would be great if frigate allowed record configuration per camera. 
For example I don't really need last full 10 days of recording for a few specific cameras, so I'd just want to have it saved motion. 

Is there a plan to make record configuration more flexible ? (set saving mode per camera, change the retain days per camera)

**Comment by NickM-27** [comment]
This is already supported, basically every setting is configurable at the camera level
